github_actions/run.sh

```diff
@@ -24,4 +24,17 @@ run_fossa()
   exit $result
 }
 
+# DEPRECATION NOTICE
+echo "******************************************************************"
+echo "  NOTICE : THIS SCRIPT HAS BEEN PULLED FROM A DEPRECATED SOURCE"
+echo "           PLEASE SEE :"
+echo "  https://github.com/mdsol/fossa_ci_scripts/blob/main/github_actions/SETUP.md"
+echo "******************************************************************"
+
+# Randomly Fail the Build to enforce adoption of new branch name
+#FAIL_BUILD=$(($RANDOM%2))
```

Comment:
I might recommend that instead of failing half of all builds we fail a smaller percentage (maybe 10%?) and then increase the percentage of failures over time until it is 100% and we delete the branch?

Reply:
Updated to 10% across all scripts